migrations/2023-12-12-150828_community_aggregates_add_local_subscribers/up.sql

```diff
@@ -0,0 +1,70 @@
+-- Couldn't find a way to put local_subscribers right after subscribers except recreating the table.
+ALTER TABLE community_aggregates
+    ADD COLUMN local_subscribers int8 NOT NULL DEFAULT 0;
+
+-- update initial value
+UPDATE
+    community_aggregates
+SET
+    local_subscribers = (
+        SELECT
+            COUNT(*)
+        FROM
+            community_follower cf
+        WHERE
+            cf.community_id = community_aggregates.community_id
+            AND (
+                SELECT
+                    local
+                FROM
+                    person
+                WHERE
+                    person.id = cf.person_id));
+
+CREATE OR REPLACE FUNCTION community_aggregates_local_subscriber_count ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    IF (TG_OP = 'INSERT') THEN
+        UPDATE
+            community_aggregates
+        SET
+            local_subscribers = local_subscribers + 1
+        FROM
+            community c
+        WHERE
+            c.id = NEW.community_id
+            AND (
```

Comment:
You shouldn't need a subquery here, check out the right way to do updates where you need to join to data: https://stackoverflow.com/a/7869611/1655478